wandb/apis/public/api.py

```diff
@@ -1157,8 +1158,14 @@ def artifact(self, name, type=None):
         if name is None:
             raise ValueError("You must specify name= to fetch an artifact.")
         entity, project, artifact_name = self._parse_artifact_path(name)
+
+        organization = ""
+        # If its an Registry artifact, the entity is an org instead
+        if is_artifact_registry_project(project):
+            parts = name.split("/")
+            organization = parts[0] if len(parts) == 3 else None
```

Comment:
I'm inclined to leave the code as is since i don't want to assign the organization variable to entity(this will be for sure a team entity since we are using default entity) if thats not an org.

Reply:
sorry, you're right. i think i just confused myself into thinking you can resolve the org name from the default entity. nevermind.